script.js

```diff
@@ -0,0 +1,145 @@
+var markers = document.body.querySelectorAll(".marker")
+var playableSpaces = document.body.querySelectorAll(".playable_square")
+var topRow = document.getElementById("top_row").children
+var middleRow = document.getElementById("middle_row").children
+var bottomRow = document.getElementById("bottom_row").children
+var counter = []
+
+function playTopLeft() {
+  counter.push(topRow[0]);
+  if(counter.length % 2 === 0){
+    topRow[0].className = "markedO"
+    markers[0].innerHTML = "O"
+  }
+  else if (counter.length % 2 !== 0){
+    topRow[0].className = "markedX"
+    markers[0].innerHTML = "X"
+  }
+  return counter.length
+}
```

Comment:
I like your use of modulo to figure out x's and o's. I'm curious though, what happens if i click the same area twice? Additionally, it might get really expensive(memory wise) to push entire dom elements into an array rather than just using an integer and incrementing it.